source_gen/pubspec.yaml

```diff
@@ -26,3 +27,31 @@ dev_dependencies:
   dart_flutter_team_lints: ^3.1.0
   term_glyph: ^1.2.0
   test: ^1.16.0
+
```

Comment:
just published these! we can drop 'em

Reply:
These are actually needed, they expose a different API to get at the new element models, but its only to help with the internal migration.